bundles/pom.xml

```diff
@@ -136,6 +131,7 @@
     <module>org.openhab.binding.electroluxappliance</module>
     <module>org.openhab.binding.elerotransmitterstick</module>
     <module>org.openhab.binding.elroconnects</module>
+    <module>org.openhab.binding.emby</module>
```

Comment:
Only this change should be here, all other (remove white lines) should be reverted)